Live-Snap/Live-Snap/Controllers/SelectPhotoViewController.swift

```diff
@@ -241,5 +251,37 @@ class SelectPhotoViewController: UIViewController, UICollectionViewDataSource, U
         containerView.removeFromSuperview()
     }
     
+    
+    func photoLibraryRequestDenied() {
+        
+        let alertController = UIAlertController (title: "Please Allow Photo Access", message: "Live-Snap needs access to your photo library to import and export images. Please click the settings button below and enable access.", preferredStyle: .alert)
```

Comment:
dont put this as an alert, the error message should be part of the view. there should be a label with the text and the go to settings button underneath. we will also need to figure out the logic for what should happen when the user goes to settings, enables it, and comes back. We should probably check for photos permission in viewdidappear